FRAuth/FRAuth/Callbacks/MultipleValuesCallback.swift

```diff
@@ -14,7 +14,7 @@ import Foundation
  MultipleValuesCallback is a base Callback implementation that has one or more user input values. Any Callback that accepts multiple values from user interaction without OpenAM's validation with policies may inherit from this class.
  */
 @objc(FRMultipleValuesCallback)
-public class MultipleValuesCallback: Callback {
+open class MultipleValuesCallback: Callback {
```

Comment:
Update year on the header for the modified classes

Reply:
Updated the header, thanks for pointing that out.